web_frontend/cloudscheduler/glintwebui/glint_views.py

```diff
@@ -673,22 +696,27 @@ def delete(request, args=None, response_code=0, message=None):
                 rc, msg, image_list = config.db_query(IMAGES, where=where)
                 if len(image_list) > 1:
                     config.db_close()
-                    return HttpResponse(json.dumps({'response_code': 1, 'message': '%s %s' % (lno(MODID), "Unable to uniquely identify target image with given parameters, %s images matched." % len(image_list))}))
+                    return HttpResponse(json.dumps({'response_code': 1, 'message': '%s %s' % (lno(MODID),
+                                                                                              "Unable to uniquely identify target image with given parameters, %s images matched." % len(
+                                                                                                  image_list))}))
                 elif len(image_list) == 0:
                     config.db_close()
-                    return HttpResponse(json.dumps({'response_code': 1, 'message': '%s %s' % (lno(MODID), "No images matched sepcified parameters, please check your request and try again.")}))
+                    return HttpResponse(json.dumps({'response_code': 1, 'message': '%s %s' % (
+                    lno(MODID), "No images matched sepcified parameters, please check your request and try again.")}))
                 else:
                     target_image = image_list[0]
 
             # we have target image, lets delete it
-            where_clause = "group_name='%s' and cloud_name='%s'" % (target_image["group_name"], target_image["cloud_name"])
-            rc, msg, cloud_list =  config.db_query(CLOUDS, where=where_clause)
+            where_clause = "group_name='%s' and cloud_name='%s'" % (
+            target_image["group_name"], target_image["cloud_name"])
+            rc, msg, cloud_list = config.db_query(CLOUDS, where=where_clause)
             cloud = cloud_list[0]
             result = delete_image(cloud, target_image["id"])
 
             if result[0] == 0:
                 # Successful delete, now delete it from csv2 database - build delete statement
-                sql ="delete from cloud_images where group_name='%s' and cloud_name='%s' and id='%s';" % (target_image["group_name"], target_image["cloud_name"], target_image["id"])
+                sql = "delete from cloud_images where group_name='%s' and cloud_name='%s' and id='%s';" % (
```

Comment:
if you're going to break up a line of code to conform to a style guide the 2nd and subsequent lines should be indented to show it's not a new line of code. This is a problem all over this file.